package.json

```diff
@@ -15,7 +15,6 @@
     "date-fns": "^2.12.0",
     "express": "^4.17.1",
     "firebase": "^7.14.0",
-    "jsdoc": "^3.6.4",
```

Comment:
Looks like this didn't get moved to `dev-dependencies`

Reply:
whoops, now it should be ok! 😅